src/MarginTrading.SettingsService.Core/MarginTrading.SettingsService.Core.csproj

```diff
@@ -15,4 +15,7 @@
     <PackageReference Include="Lykke.RabbitMqBroker" Version="7.0.0" />
     <PackageReference Include="Lykke.SettingsReader" Version="2.3.1" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\MarginTrading.SettingsService.Contracts\MarginTrading.SettingsService.Contracts.csproj" />

```

Comment:
Not the best idea to make core dependent on contract...  This reference is used only in LegalEntityExtensions and it should be moved to SettingsService project I think.